<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0" stdsetdef="1">
  <author></author>
  <comment></comment>
  <exportmacro></exportmacro>
  <class>KEditScheduleDlgDecl</class>
  <widget class="QDialog" name="KEditScheduleDlgDecl">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>798</width>
        <height>640</height>
      </rect>
    </property>
    <property name="windowTitle">
      <string>Edit Scheduled transaction</string>
    </property>
    <property name="sizeGripEnabled">
      <bool>true</bool>
    </property>
    <layout class="QVBoxLayout">
      <item>
        <layout class="QHBoxLayout">
          <item>
            <layout class="QHBoxLayout">
              <item>
                <widget class="QLabel" name="textLabel1">
                  <property name="text">
                    <string>Schedule name:</string>
                  </property>
                  <property name="wordWrap">
                    <bool>false</bool>
                  </property>
                </widget>
              </item>
              <item>
                <widget class="KLineEdit" name="m_nameEdit">
                  <property name="focusPolicy">
                    <enum>Qt::StrongFocus</enum>
                  </property>
                </widget>
              </item>
            </layout>
          </item>
          <item>
            <layout class="QHBoxLayout">
              <item>
                <widget class="QLabel" name="textLabel3">
                  <property name="text">
                    <string>Frequency:</string>
                  </property>
                  <property name="wordWrap">
                    <bool>false</bool>
                  </property>
                </widget>
              </item>
              <item>
                <widget class="KIntNumInput" name="m_frequencyNoEdit">
                  <property name="maximum">
                    <number>999</number>
                  </property>
                  <property name="minimum">
                    <number>1</number>
                  </property>
                  <property name="toolTip" stdset="0">
                    <string>Number of selected periods between entries</string>
                  </property>
                  <property name="focusPolicy">
                    <enum>Qt::StrongFocus</enum>
                  </property>
                </widget>
              </item>
              <item>
                <widget class="KMyMoneyOccurrencePeriodCombo" name="m_frequencyEdit"/>
              </item>
            </layout>
          </item>
        </layout>
      </item>
      <item>
        <widget class="QGroupBox" name="m_paymentInformation">
          <property name="title">
            <string>Payment information</string>
          </property>
          <layout class="QVBoxLayout" name="m_paymentInformationLayout">
            <item>
              <layout class="QHBoxLayout">
                <item>
                  <widget class="QLabel" name="textLabel1_3">
                    <property name="text">
                      <string>Payment method</string>
                    </property>
                    <property name="wordWrap">
                      <bool>false</bool>
                    </property>
                  </widget>
                </item>
                <item>
                  <widget class="KMyMoneyGeneralCombo" name="m_paymentMethodEdit">
                    <property name="sizePolicy">
                      <sizepolicy>
                        <hsizetype>3</hsizetype>
                        <vsizetype>0</vsizetype>
                        <horstretch>0</horstretch>
                        <verstretch>0</verstretch>
                      </sizepolicy>
                    </property>
                  </widget>
                </item>
                <item>
                  <spacer name="spacer4">
                    <property name="sizeHint">
                      <size>
                        <width>61</width>
                        <height>21</height>
                      </size>
                    </property>
                    <property name="sizeType">
                      <enum>QSizePolicy::Expanding</enum>
                    </property>
                    <property name="orientation">
                      <enum>Qt::Horizontal</enum>
                    </property>
                  </spacer>
                </item>
              </layout>
            </item>
            <item>
              <widget class="Line" name="line1">
                <property name="frameShape">
                  <enum>QFrame::HLine</enum>
                </property>
                <property name="frameShadow">
                  <enum>QFrame::Sunken</enum>
                </property>
              </widget>
            </item>
            <item>
              <widget class="KMyMoneyRegister::Register" name="m_register">
                <property name="rowCount">
                  <number>0</number>
                </property>
                <property name="columnCount">
                  <number>12</number>
                </property>
                <column>
                  <property name="text">
                    <string>Security</string>
                  </property>
                </column>
                <column>
                  <property name="text">
                    <string>Details</string>
                  </property>
                </column>
                <column>
                  <property name="text">
                    <string>C</string>
                  </property>
                </column>
                <column>
                  <property name="text">
                    <string>Payment</string>
                  </property>
                </column>
                <column>
                  <property name="text">
                    <string>Deposit</string>
                  </property>
                </column>
                <column>
                  <property name="text">
                    <string>Quantity</string>
                  </property>
                </column>
                <column>
                  <property name="text">
                    <string>Price</string>
                  </property>
                </column>
                <column>
                  <property name="text">
                    <string>Value</string>
                  </property>
                </column>
                <column>
                  <property name="text">
                    <string>Balance</string>
                  </property>
                </column>
              </widget>
            </item>
            <item>
              <widget class="KMyMoneyTransactionForm::TransactionForm" name="m_form">
                <property name="rowCount">
                  <number>5</number>
                </property>
                <property name="columnCount">
                  <number>1</number>
                </property>
              </widget>
            </item>
          </layout>
        </widget>
      </item>
      <item>
        <widget class="QGroupBox" name="groupBox2">
          <property name="title">
            <string>Options</string>
          </property>
          <layout class="QVBoxLayout">
            <item>
              <layout class="QGridLayout">
                <item row="1" column="1">
                  <widget class="KComboBox" name="m_weekendOptionEdit">
                    <item>
                      <property name="text">
                        <string>Do nothing</string>
                      </property>
                    </item>
                    <item>
                      <property name="text">
                        <string>Change the date to the previous processing day</string>
                      </property>
                    </item>
                    <item>
                      <property name="text">
                        <string>Change the date to the next processing day</string>
                      </property>
                    </item>
                  </widget>
                </item>
                <item row="1" column="0">
                  <widget class="QLabel" name="textLabel1_2">
                    <property name="text">
                      <string>If this schedule occurs on a non-processing day:</string>
                    </property>
                    <property name="wordWrap">
                      <bool>false</bool>
                    </property>
                  </widget>
                </item>
              </layout>
            </item>
            <item>
              <layout class="QHBoxLayout">
                <item>
                  <widget class="QCheckBox" name="m_estimateEdit">
                    <property name="text">
                      <string>The amount is an estimate because it varies for each payment</string>
                    </property>
                  </widget>
                </item>
                <item>
                  <widget class="KIntSpinBox" name="m_variation">
                    <property name="enabled">
                      <bool>false</bool>
                    </property>
                    <property name="value">
                      <number>10</number>
                    </property>
                  </widget>
                </item>
                <item>
                  <spacer name="spacer5">
                    <property name="sizeHint">
                      <size>
                        <width>21</width>
                        <height>20</height>
                      </size>
                    </property>
                    <property name="sizeType">
                      <enum>QSizePolicy::Expanding</enum>
                    </property>
                    <property name="orientation">
                      <enum>Qt::Horizontal</enum>
                    </property>
                  </spacer>
                </item>
              </layout>
            </item>
            <item>
              <widget class="QCheckBox" name="m_lastDayInMonthEdit">
                <property name="text">
                  <string>Process this schedule always at the last day of a month</string>
                </property>
              </widget>
            </item>
            <item>
              <widget class="QCheckBox" name="m_autoEnterEdit">
                <property name="text">
                  <string>Enter this schedule into the register automatically when it is due</string>
                </property>
              </widget>
            </item>
            <item>
              <widget class="QCheckBox" name="m_endSeriesEdit">
                <property name="text">
                  <string>This schedule will end at some time</string>
                </property>
              </widget>
            </item>
            <item>
              <widget class="QFrame" name="m_endOptionsFrame">
                <property name="frameShape">
                  <enum>QFrame::NoFrame</enum>
                </property>
                <property name="frameShadow">
                  <enum>QFrame::Raised</enum>
                </property>
                <layout class="QHBoxLayout">
                  <property name="margin">
                    <number>0</number>
                  </property>
                  <item>
                    <spacer name="Spacer3_2_2">
                      <property name="sizeHint">
                        <size>
                          <width>20</width>
                          <height>20</height>
                        </size>
                      </property>
                      <property name="sizeType">
                        <enum>QSizePolicy::Fixed</enum>
                      </property>
                      <property name="orientation">
                        <enum>Qt::Horizontal</enum>
                      </property>
                    </spacer>
                  </item>
                  <item>
                    <widget class="QLabel" name="m_endLabel1">
                      <property name="text">
                        <string>Number of transactions remaining:</string>
                      </property>
                      <property name="wordWrap">
                        <bool>false</bool>
                      </property>
                    </widget>
                  </item>
                  <item>
                    <widget class="KIntNumInput" name="m_RemainingEdit">
                      <property name="minimum">
                        <number>0</number>
                      </property>
                      <property name="maximum">
                        <number>9999</number>
                      </property>
                    </widget>
                  </item>
                  <item>
                    <widget class="QLabel" name="m_endLabel2">
                      <property name="text">
                        <string>Date of final transaction:</string>
                      </property>
                      <property name="wordWrap">
                        <bool>false</bool>
                      </property>
                    </widget>
                  </item>
                  <item>
                    <widget class="kMyMoneyDateInput" name="m_FinalPaymentEdit">
                      <property name="focusPolicy">
                        <enum>Qt::StrongFocus</enum>
                      </property>
                    </widget>
                  </item>
                </layout>
              </widget>
            </item>
          </layout>
        </widget>
      </item>
      <item>
        <spacer name="spacer12">
          <property name="sizeHint">
            <size>
              <width>20</width>
              <height>16</height>
            </size>
          </property>
          <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="orientation">
            <enum>Qt::Vertical</enum>
          </property>
        </spacer>
      </item>
      <item>
        <layout class="QHBoxLayout">
          <property name="margin">
            <number>0</number>
          </property>
          <property name="spacing">
            <number>6</number>
          </property>
          <item>
            <widget class="KPushButton" name="buttonHelp">
              <property name="text">
                <string>Help</string>
              </property>
            </widget>
          </item>
          <item>
            <spacer name="Spacer1_2">
              <property name="sizeHint">
                <size>
                  <width>280</width>
                  <height>0</height>
                </size>
              </property>
              <property name="sizeType">
                <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="orientation">
                <enum>Qt::Horizontal</enum>
              </property>
            </spacer>
          </item>
          <item>
            <widget class="KPushButton" name="buttonOk">
              <property name="text">
                <string>OK</string>
              </property>
              <property name="autoDefault">
                <bool>true</bool>
              </property>
              <property name="default">
                <bool>true</bool>
              </property>
            </widget>
          </item>
          <item>
            <widget class="KPushButton" name="buttonCancel">
              <property name="text">
                <string>Cancel</string>
              </property>
            </widget>
          </item>
        </layout>
      </item>
    </layout>
  </widget>
  <layoutdefault spacing="6" margin="11"/>
    <customwidgets>
    <customwidget>
      <class>KComboBox</class>
      <extends>QComboBox</extends>
      <header location="global">kcombobox.h</header>
    </customwidget>
  <customwidget>
   <class>KIntNumInput</class>
   <extends>QWidget</extends>
   <header>knuminput.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KIntSpinBox</class>
   <extends>QSpinBox</extends>
   <header>knuminput.h</header>
  </customwidget>
  <customwidget>                                                    
   <class>KMyMoneyGeneralCombo</class>                                
   <extends>QWidget</extends>                                       
   <header>kmymoneymvccombo.h</header>
   <container>1</container>                                         
  </customwidget>  
  <customwidget>
   <class>KMyMoneyOccurrencePeriodCombo</class>
   <extends>QWidget</extends>
   <header>kmymoneymvccombo.h</header>
   <container>1</container>
  </customwidget>

  </customwidgets>
  <connections>
    <connection>
      <sender>buttonOk</sender>
      <signal>clicked()</signal>
      <receiver>KEditScheduleDlgDecl</receiver>
      <slot>accept()</slot>
    </connection>
    <connection>
      <sender>m_endSeriesEdit</sender>
      <signal>toggled(bool)</signal>
      <receiver>m_endOptionsFrame</receiver>
      <slot>setEnabled(bool)</slot>
    </connection>
    <connection>
      <sender>buttonCancel</sender>
      <signal>clicked()</signal>
      <receiver>KEditScheduleDlgDecl</receiver>
      <slot>reject()</slot>
    </connection>
    <connection>
      <sender>m_estimateEdit</sender>
      <signal>toggled(bool)</signal>
      <receiver>m_variation</receiver>
      <slot>setEnabled(bool)</slot>
    </connection>
  </connections>
</ui>
